### PR TITLE
Update README Screenshots

### PR DESCRIPTION
**Pull request by matt852**
# Closes: #1667
# What's Changed

When I view the current screenshots on the README landing page, I do not feel like they accurately show all of Nautobot's capabilities, benefits, or really differentiate it from Netbox. The current screenshots are from Nautobot v1.0.0b1, and there are significant changes since then that I think should be featured.

I went ahead and updated them to more accurately reflect Nautobot's current state of development, as well as highlight some great new features it offers. The updated screenshots are from v1.2.9 and show base functionality as well as some plugin benefits.

Note: This was originally opened as PR #1668, but I closed that and resubmitted the PR after some merging issues in the branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,15 +22,19 @@ Questions? Comments? Start by perusing our [GitHub discussions](https://github.c
 
 ## Screenshots
 
-![Screenshot of main page](https://raw.githubusercontent.com/nautobot/nautobot/develop/nautobot/docs/media/screenshot1.png "Main page")
+![Screenshot of main page](https://raw.githubusercontent.com/nautobot/nautobot/develop/nautobot/docs/media/ss_main_page.png "Main page")
```
Comment (glennmatthews) [inline README.md:25]: There's a decent amount of blank space in the lower half of this image - possibly resize the browser window to eliminate it and redo the screenshot?

**Review by glennmatthews — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (matt852) [reply README.md:25]: Done

**Review by bryanculver — COMMENTED** [review]
@matt852 Is there a reason this file has excessive whitespace at the bottom: https://github.com/nautobot/nautobot/pull/1671/files?short_path=ba51f96#diff-ba51f968107f61036b3c956b27512cf834573d7a21ba593ebed9dbc882d972c2

**Comment by matt852** [comment]
> @matt852 Is there a reason this file has excessive whitespace at the bottom: https://github.com/nautobot/nautobot/pull/1671/files?short_path=ba51f96#diff-ba51f968107f61036b3c956b27512cf834573d7a21ba593ebed9dbc882d972c2

@bryanculver The reason is I can't multi-task while cropping images properly.  My bad.

**Pull request merged**